sample_aws_container/src/sample.py

```diff
@@ -0,0 +1,18 @@
+def import_lightrun():
+    try:
+        import lightrun
+        lightrun.enable(com_lightrun_server='https://app.lightrun.com/company/success', company_key='b33aaabc-2ef0-4bcd-9c09-e3de48ac90fe')
+    except ImportError as e:
+        print("Error importing Lightrun: ", e)
+
+def start_fibonacci(n):
+    if n in {0, 1}:
+        return n
+    return start_fibonacci(n - 1) + start_fibonacci(n - 2)
+
+def main(event, context):
+    import_lightrun()
+    num = event['num']
+
+    print("Calculating Fibonacci of {}...".format(num))
+    start_fibonacci(num)
```

Comment:
Don't you want to print the result? or return it?

Reply:
this should be without prints/logs as the idea is to run the app and play with lightrun for demo

Reply:
I don't think this app makes sense :/

Reply:
why not..? We have just to run lambda with any app, that will last long to demonstrate lightrun.